src/dotnet/commands/dotnet-tool/update/ToolUpdateCommand.cs

```diff
@@ -112,8 +124,11 @@ public override int Execute()
                         new PackageLocation(nugetConfig: GetConfigFile(), additionalFeeds: _additionalFeeds),
                         packageId: _packageId,
                         targetFramework: _framework,
+                        versionRange: versionRange,
                         verbosity: _verbosity);
 
+                    EnsureVersionIsHigher(oldPackageNullable, newInstalledPackage);
```

Comment:
I think we should have a check above, before we enter the transaction scope, that checks if we have an old package and if the old package is the requested version, then gracefully complete with a message informing them that the requested version is already installed.

What do you think, @wli3?

Reply:
I'm also going to resolve this suggestion too since the way we're handling the version seems to indicate that `--version 1.2.3.4` will parse as a minimum version and not exact, thus we can't make this check anyway.